### PR TITLE
Add new version 3.8.3

### PR DESCRIPTION
**Pull request by maltesander**
# Description

- added removed and deprecated versions docs
- bumped zk version to 3.8.3

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [ ] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
- [x] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: docs/modules/zookeeper/examples/getting_started/code/getting_started.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ zkCli_ls() {
 # tag::zkcli-ls[]
 kubectl run my-pod \
   --stdin --tty --quiet --restart=Never \
-  --image docker.stackable.tech/stackable/zookeeper:3.8.0-stackable0.8.0 -- \
+  --image docker.stackable.tech/stackable/zookeeper:3.8.3-stackable0.8.0 -- \
```
Comment (razvan) [inline docs/modules/zookeeper/examples/getting_started/code/getting_started.sh:68]:
```suggestion
  --image docker.stackable.tech/stackable/zookeeper:3.8.3-stackable0.0.0-dev -- \
```

File: docs/modules/zookeeper/examples/getting_started/code/getting_started.sh.j2 [hunk 2]
```diff
@@ -65,7 +65,7 @@ zkCli_ls() {
 # tag::zkcli-ls[]
 kubectl run my-pod \
   --stdin --tty --quiet --restart=Never \
-  --image docker.stackable.tech/stackable/zookeeper:3.8.0-stackable0.8.0 -- \
+  --image docker.stackable.tech/stackable/zookeeper:3.8.3-stackable0.8.0 -- \
```
Comment (razvan) [inline docs/modules/zookeeper/examples/getting_started/code/getting_started.sh.j2:68]:
```suggestion
  --image docker.stackable.tech/stackable/zookeeper:3.8.3-stackable0.0.0-dev -- \
```

**Review by razvan — COMMENTED** [review]
some typos

**Comment by maltesander** [comment]
test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/zookeeper-operator-it-custom/65/

**Review by razvan — APPROVED** [review]

**Pull request merged**